src/main/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepository.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.blobrouter.data;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.blobrouter.data.model.EnvelopeSummary;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+
+@Repository
+public class ReportRepository {
+
+    private static final String BULKSCAN_CONTAINER_NAME = "bulkscan";
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final EnvelopeSummaryMapper mapper;
+
+    public ReportRepository(
+        NamedParameterJdbcTemplate jdbcTemplate,
+        EnvelopeSummaryMapper mapper
+    ) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.mapper = mapper;
+    }
+
+    public List<EnvelopeSummary> find(Instant from, Instant to) {
```

Comment:
perhaps:

```suggestion
    public List<EnvelopeSummary> getEnvelopeSummary(Instant from, Instant to) {
```

?

Reply:
do not mind

Reply:
repository is now `ReportRepository` - should call names more specific than `find`. Imagine the usage:

```java

@Autowired
private ReportRepository reportRepository;

var findings = reportRepository->find(from, to);
```

🤷‍♂ find reports i guess 😄